doc/source/schemas/sequenceAnnotationmethods.rst

```diff
@@ -0,0 +1,457 @@
+SequenceAnnotationMethods
```

Comment:
Okay, I realize that I might be fixating, but one more observation in this regard (until the next one):

doc/source/api/sequence_annotations.rst sayeth:

> The central object of the GA4GH Sequence Annotation API is a Feature.

I think this underscores the point I'm trying to make. If the core concept is a Feature, then why isn't this the Sequence Feature API?